Python/optimizer_bytecodes.c

```diff
@@ -449,6 +449,14 @@ dummy_func(void) {
         }
     }
 
+    op(_LOAD_ATTR, (owner -- attr, self_or_null if (oparg & 1))) {
+        (void)owner;
+        OUT_OF_SPACE_IF_NULL(attr = sym_new_not_null(ctx));
+        if (oparg & 1) {
+            OUT_OF_SPACE_IF_NULL(self_or_null = sym_new_unknown(ctx));
```

Comment:
At some point in the future we will be able to tell whether the attribute is a method or not, and hence we will be able to set it to either `null` or `not_null` -- but not today, right?

Reply:
No not for _LOAD_ATTR - this opcode is special, it means we dont know anything and it's only decideable at runtime.